setup.py

```diff
@@ -133,11 +133,12 @@ def get_extensions():
                 nvcc_flags = []
             else:
                 nvcc_flags = nvcc_flags.split(' ')
+            nvcc_flags.append('-O3')
```

Comment:
On Windows, it should be `-O2`. `-O3` is not a valid flag.

Reply:
Thanks for the heads up!

By looking at the compilation logs on Windows, it seems that we have `/Ox` that appeared, which would imply that the `-O3` is working ok on Windows somehow?

Reply:
`/Ox` is added by disutils. It is an optimization level between /O1 and /O2.

Reply:
I added `/O2` for both nvcc and cxx, let me know if this isn't what you were thinking about

Reply:
Actually, nvcc accepts `-O3` on Windows. But they say that it is already added as a default. https://forums.developer.nvidia.com/t/how-to-do-o3-optimization-in-visual-studio-for-cuda-code/38843/5.

Reply:
Thanks @peterjc123 

One thing I'm wondering, I couldn't find any reference to `/O2` in the args passed to the compiler in https://app.circleci.com/pipelines/github/pytorch/vision/3113/workflows/1a383c11-0f1b-4d90-859d-551a136fb9ed/jobs/172391/steps

Is it being taken into account on Windows?

Reply:
@fmassa That's weird. I'll look into that.

Reply:
Apparently, https://github.com/pytorch/vision/pull/2378/files#diff-2eeaed663bd0d25b7e608891384b7298L125 is the cause. For CPU tests, that if condition will not be met, so we won't run those lines. I've checked that `/O2` is in GPU unit tests so there is nothing weird here.